photon-ml/src/main/scala/com/linkedin/photon/ml/avro/AvroUtils.scala

```diff
@@ -91,27 +92,28 @@ object AvroUtils {
     NameAndTerm(name, term)
   }
 
-  protected[avro] def modelToBayesianLinearModelAvro(
-      model: Coefficients,
+  protected[avro] def convertCoefficientsToBayesianLinearModelAvroRecord(
+      coefficients: Coefficients,
       modelId: String,
       intToNameAndTermMap: Map[Int, NameAndTerm]): BayesianLinearModelAvro = {
 
-    val meansAvros = vectorToArrayOfNameTermValueAvros(model.means, intToNameAndTermMap)
-    val variancesAvrosOption = model.variancesOption.map(variances => vectorToArrayOfNameTermValueAvros(variances,
-      intToNameAndTermMap))
+    val meansAvros = vectorToArrayOfNameTermValueAvros(coefficients.means, intToNameAndTermMap)
+    val variancesAvrosOption = coefficients.variancesOption.map(variances =>
+      vectorToArrayOfNameTermValueAvros(variances, intToNameAndTermMap)
+    )
```

Comment:
Minor: it's debatable whether this is actually an improvement, but I'm a big fan of point-free style:

```scala
val variancesAvrosOption = coefficients.variancesOption.map(
  vectorToArrayOfNameTermValueAvros(_, intToNameAndTermMap))
```